challenges/objects-arrays.js

```diff
@@ -50,7 +95,14 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
+for (let i = 0 ; i < graduates.length; i++){
+    function doSomething(){
+      universities.push(graduates[i].university);
+      universities.sort();
```

Comment:
I would only sort the array after everything has been added to it, that way the computer has to do less stuff and your program runs faster